ocaml/runtime/minor_gc.c

```diff
@@ -449,7 +449,16 @@ static void oldify_mopup (struct oldify_state* st, int do_ephemerons)
       oldify_one (st, f, Op_val (new_v));
     }
 
-    for (i = 1; i < scannable_wosize; i++){
+    i = 1;
+
+    if(Tag_val(new_v) == Closure_tag) {
```

Comment:
Max asked me whether it's fine to use `Tag_val` to get the tag without going through `get_header_val` to ensure that nobody else is writing the header, and I strongly suspect it's fine: other places in this file use `Tag_val`.